tests/integration/integration-test.py

```diff
@@ -46,8 +46,8 @@ def test_get_notes_path(self, endpoint='/notes'):
             self.assertEqual(actual_note_id, note_id)
```

Comment:
In addition, I think we can also check the student ID is also matched with the prefix of the note ID. What do you think? Or do you think there is additional stuff we can test with?

Reply:
when i started implementing this function i had that in mind, but then i realized that we don't have the student id before we make the request so I don't think its possible to check the student ID from making a request with the note ID

Reply:
Why not? You should be able to compare the [`studentId`](https://github.com/osu-mist/notes-api/blob/master/openapi.yaml#L221-L222) field with the prefix of the `noteId`, right?

Reply:
right! i forgot that the first part of the note id is the student id